UserService/app/src/main/java/com/bookmyshow/userservice/dto/PermissionRequestDTO.java

```diff
@@ -0,0 +1,46 @@
+package com.bookmyshow.userservice.dto;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import javax.validation.constraints.NotBlank;
+
+@JsonPropertyOrder({
+        "permissionCode",
+        "permissionDescription"
+})
+public class PermissionRequestDTO {
+
+    @NotBlank
+    private String permissionCode;
```

Comment:
we can name this variable as "code" since we are already into a Permission DTO class